javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -558,6 +560,9 @@ public static boolean hasInheritedMeta(AnnotationMirror anno) {
                 if (elementType.name().contentEquals("MODULE")) {
                     return EnumSet.noneOf(ElementKind.class);
```

Comment:
Regarding your second question: https://github.com/typetools/checker-framework/commit/3ef0af3636c7f6662c638699dddc6a48ce1378f5 added this work-around.
I'm not quite sure why we need it, as we use at least a Java 9 compiler, so using MODULE should always work.
I agree that using `ElementKind.valueOf("MODULE")` should also work here.
Could you try that as a separate PR?